File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        minDis = 100000
        for i in range(len(nums)):
            if nums[i] == target:
                dis = abs(i-start)
                if dis < minDis:
                    minDis = dis
        return minDis

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int minDis = INT_MAX;
        for(int i = 0; i<nums.size(); i++){
            if(nums[i] == target){
                minDis = min(minDis, abs(i-start));
            }
        }
        return minDis;
    }
};